packages/contracts-bedrock/src/L1/L1CrossDomainMessenger.sol

```diff
@@ -8,7 +8,7 @@ import { ISemver } from "src/universal/ISemver.sol";
 import { SuperchainConfig } from "src/L1/SuperchainConfig.sol";
 import { SystemConfig } from "src/L1/SystemConfig.sol";
 
-/// @custom:proxied
+/// @custom:proxied true
```

Comment:
Had to be updated to `@custom:proxied true` because the Solidity compiler generates natspec assuming there's a value field, which resulted in an output like `"proxied": "@title L1CrossDomainMessenger"` (eating the line below it).